### PR TITLE
Black Lives Matter banner

### PR DESCRIPTION
**Pull request by cpojer**
Supporting a movement that is important and will hopefully make a difference.

Screen Shot 2020-06-04 at 21 44 41

Screen Shot 2020-06-04 at 1 55 52 PM

### CODE REVIEW

File: website/static/css/homepage/HeaderHero.css [hunk 1]
```diff
@@ -49,6 +49,34 @@
...
+  max-width: 768px;
+}
+
+.HeaderHero .announcement-inner a {
+  text-decoration: underline;
+}
+
+.HeaderHero .announcement-inner a:hover {
+  color: $brand;
+}
```
Comment (rickhanlonii) [inline website/static/css/homepage/HeaderHero.css:78]:
```suggestion
}

/* End announcement banner override */
```

**Comment by JoshuaGross** [comment]
<3

**Comment by kelset** [comment]
♥♥♥♥♥♥

**Comment by ianjennings** [comment]
@cpojer @yungsters :)

**Comment by lLenn** [comment]
Can this be removed, please? I don't think a coding repository is the appropriate place to do political activism.

**Pull request merged**